content/source/docs/cloud/sentinel/import/tfplan.html.md

```diff
@@ -354,6 +354,8 @@ Some examples of multi-level access are below:
   `tfplan.resources.aws_instance`, with names being the next level down, and so
   on.
 
+~> When [resource targeting](/docs/commands/plan.html#resource-targeting) is in effect, `tfplan.resources` will only include the resources specified as targets for the run. This may lead to unexpected outcomes if a policy expects a resource to always exist. To prohibit targeted runs altogether, use [`tfrun.target_addrs`](/docs/cloud/sentinel/import/tfrun.html#value-target_addrs).
```

Comment:
"a policy expects a resource to always exist" - maybe "a policy expects a resource to be present in the plan"

Maybe that is less clear, but "to always exist" made me think it was talking about the underlying resource actually existing vs the value being handy for sentinel.

Reply:
Good point!